frontend/src/metabase/query_builder/components/DataSelector/saved-entity-picker/utils.js

```diff
@@ -1,9 +1,9 @@
-export const findCollectionByName = (collections, name) => {
+export const findCollectionByName = (collections, collectionId) => {
```

Comment:
Let's rename this function to `findCollectionById` as it takes a collection id.

```suggestion
export const findCollectionById = (collections, collectionId) => {
```

Reply:
LGTM, fixed